### PR TITLE
US7206: Style View Source Code Window

### PR DESCRIPTION
**Pull request by ozhangd2l**
added daylight styles for source code view dialog (and dialogs in tinymce in general). Also includes Alex's styling for the color picker.

### CODE REVIEW

File: js/tinymce/skins/lightgray/FloatPanel.less [hunk 1]
```diff
@@ -81,4 +81,87 @@
...
 }
-// End of D2L
\ No newline at end of file
+
+// Conform OK button style to daylight
+.@{prefix}-floatpanel .@{prefix}-primary.@{prefix}-btn {
+	left: 0 !important;
+	top: initial !important;
+	bottom: 0 !important;
+	border: 1px solid #1b5295;
```
Comment (mdgbayly) [inline js/tinymce/skins/lightgray/FloatPanel.less]: Ideally we'd use LESS variables for some of these colors to improve maintainability?

File: js/tinymce/skins/lightgray/FloatPanel.less [hunk 2]
```diff
@@ -81,4 +81,87 @@
...
+		letter-spacing: .02rem;
+		line-height: 1rem;
+		width: 4rem;
+
+		span.@{prefix}-txt {
+			display: none;
+		}
+
+		&:after {
+			content: "Save";
```
Comment (mdgbayly) [inline js/tinymce/skins/lightgray/FloatPanel.less]: What is the purpose of the 'Save' value here?

**Comment by mdgbayly** [comment]
@ozhangd2l looks good

File: js/tinymce/skins/lightgray/FloatPanel.less (hunk 2 above)
Comment (alexli01) [inline js/tinymce/skins/lightgray/FloatPanel.less]: Its for change the OK button to SAVE button

Comment (mdgbayly) [inline js/tinymce/skins/lightgray/FloatPanel.less]: Is there a reason why we are doing that in CSS though ? As eventually we have to localize tinymce and I don't think we'll be able to do that with stuff in css.  If this is just about a label change, I'd rather we defer until we do localization.

Comment (ozhangd2l) [inline js/tinymce/skins/lightgray/FloatPanel.less]: hmmmm sure i'll change it back to the original then

**Pull request merged**